Add note on mapping limitation (#4929)
segmentio/segment-docs: src/connections/destinations/catalog/eloqua/index.md

in Eloqua and a property called `AccountType` in your Segment event, the
mapping will get handled.

> info "Track event mapping limitations"

> > Each track event can only be mapped to a single custom object; mapping a single track event to multiple custom objects is not supported.

Reviewer: ```suggestion
> Each `track` event can only be mapped to a single custom object. Segment doesn't support mapping a single `track` event to multiple custom objects.
```
